src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -197,6 +197,187 @@ The CSS approach, by keeping specificity low on base class properties and resett
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+{{/l-flex}}
+<br>
+<h3>
+  Column gap using <code>.pf-m-column-gap</code>
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-column-gap-sm"}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+{{/l-flex}}
+<br>
+<h3>
+  Gap inherits, but can be overridden when nested - <code>.pf-l-gap-xl > .pf-l-gap-xs</code>
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-gap-xl"}}
+  {{#> l-flex l-flex--modifier="pf-m-gap-xs"}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+  {{#> l-flex}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+{{/l-flex}}
+<br>
+
+<h3>
+  Row gap with individually spaced items - <code>.pf-m-row-gap-xs > pf-m-spacer-{xs,sm,md,lg,xl,2xl,3xl}</code>. Using <code>.pf-m-spacer</code> is not recommended with column gap.
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-row-gap-xs"}}
+  {{#> l-flex-item}}
+    Item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Item
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-none"}}
+    Item - none
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-xs"}}
+    Item - xs
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-sm"}}
+    Item - sm
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-md"}}
+    Item - md
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-lg"}}
+    Item - lg
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-xl"}}
+    Item - xl
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-2xl"}}
+    Item - 2xl
+  {{/l-flex-item}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-3xl"}}
+    Item - 3xl
+  {{/l-flex-item}}
+{{/l-flex}}
+```
+### Usage
+Use gap modifiers to space flex items evenly. If individual spacing of items along the main axis is needed, the [flex spacer system](#spacer) can be used instead of row gap.
+
+Using the spacer system (`.pf-m-spacer` or `.pf-m-item-space-items`) with gap or column gap is not recommended, as in some cases, it could cause elements to extend past the parent's boundary along the main axis.
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-m-gap-{none, xs, sm, md, lg, xl, 2xl}{-on-[breakpoint]}`, `.pf-m-row-gap-{none, xs, sm, md, lg, xl, 2xl}{-on-[breakpoint]}`, `.pf-m-column-gap-{none, xs, sm, md, lg, xl, 2xl}{-on-[breakpoint]}` | `.pf-l-flex` | Creates a gap, row gap, or column gap. |
```

Comment:
I think these should be unique rows in the table for `gap`, `row-gap`, and `column-gap`.

Also since you don't have to specify a size, I think the class suffix should be `.pf-m-gap{-{none, xs, sm, md, lg, xl, 2xl}{-on-[breakpoint]}}`, or something like that, just denoting that `.pf-m-[row/column-]gap` can be used on its own. @mattnolting wdyt, I feel like you know the curly vs regular braces syntax better than I do